relationships/synthesis/INFRA-HOST-to-MLOPS-MACHINE_LEARNING_MODEL.yml

```diff
@@ -0,0 +1,65 @@
+relationships:
+  - name: mlEndpointHostsModelFromEndpointName
+    version: "1"
+    origins: 
+      - APM Metrics
+    conditions:
+      - attribute: "aws..aws.sagemaker.endpoints.EndpointName"
+        present: true
+      - attribute: eventType
+        anyOf: [ "MetricRaw" ]
+      - attribute: aws.Namespace
+        anyOf: [ "/aws/sagemaker/Endpoints" ]
+    relationship:
+      expires: P75M
+      relationshipType: HOSTS
+      source:
+        extractGuid:
+          attribute: entity.guid
```

Comment:
this is missing the entity.type attribute right?

Reply:
right, changed